test/rubygems/test_require.rb

```diff
@@ -466,8 +466,7 @@ def test_second_gem_require_does_not_resolve_path_manually_before_going_through_
   end
 
   def test_realworld_default_gem
-    testing_ruby_repo = !ENV["GEM_COMMAND"].nil?
-    pend "this test can't work under ruby-core setup" if testing_ruby_repo || java_platform?
+    omit "this test can't work under ruby-core setup" if testing_ruby_repo?
```

Comment:
Shouldn't this be?
```suggestion
    omit "this test can't work under ruby-core setup" if testing_ruby_repo? || java_platform?
```

Reply:
I realized the test passes just fine on jruby now, so I figured I would remove the condition, specially since the text only references a ruby-core setup, but nothing about jruby.